modules/gameSearch/src/main/Query.scala

```diff
@@ -92,33 +92,33 @@ object Query {
       60 * 150,
       60 * 180
     ),
-    _ / 60,
-    "%d minute{s}"
+    i => trans.nbMinutes.pluralSame(i / 60).render
   ).toList
 
-  val clockIncs =
-    options(List(0, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 90, 120, 150, 180), "%d second{s}").toList
+  def clockIncs(implicit lang: Lang) =
+    options(
+      List(1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 90, 120, 150, 180),
```

Comment:
0 is gone?

Reply:
Oh I think I see your point: 0 seconds increment is different than no increment specified. 

Oups, I'll fix that

Reply:
I'm struggling with translating '0 second' ... it always comes plural (0 seconds') ... do you need a translation just for that ?

Reply:
0 is plural in english. It's language dependant.

Reply:
done